webui/common/src/components/Stacks/Stacks.test.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+import { configure, shallow, ShallowWrapper } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { createBrowserHistory, History, LocationState } from 'history';
+import React from 'react';
+import sinon from 'sinon';
+import { AllStacksProps, Stacks } from './Stacks';
+import { initialState } from '../../../../master/src/store';
+import { createAlertErrors } from '../../utilities';
+
+configure({ adapter: new Adapter() });
+
+describe('Stacks-related', () => {
```

Comment:
sorry my previous comment was unclear, i didn't mean to say it should be renamed to "Stacks-related". You can just call it "Stacks test".

Reply:
done